OutboundCallReminder/src/main/java/com/communication/outboundcallreminder/EventHandler/EventDispatcher.java

```diff
@@ -0,0 +1,99 @@
+package com.communication.outboundcallreminder.EventHandler;
+
+import com.azure.communication.callingserver.models.events.*;
+import com.azure.core.models.CloudEvent;
+import com.azure.core.util.BinaryData;
+import java.util.*;
+
+public class EventDispatcher {
+    private static EventDispatcher instance = null;
+    private Hashtable<String, NotificationCallback> NotificationCallback;
+
+    EventDispatcher() {
+        NotificationCallback = new Hashtable<String, NotificationCallback>();
+    }
+
+    /// <summary>
+    /// Get istace of EventDispatcher
+    /// </summary>
+    public static EventDispatcher GetInstance() {
+        if (instance == null) {
+            instance = new EventDispatcher();
+        }
+        return instance;
+    }
+
+    public Boolean Subscribe(String eventType, String eventKey, NotificationCallback notificationCallback) {
```

Comment:
Lets rename method name according to java naming guideline. 
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html

It should be 'subscribe()'

Reply:
please fix method names in other parts of the codebase